src/components/ui/carousel.tsx

```diff
@@ -0,0 +1,224 @@
+import * as React from 'react';
+import useEmblaCarousel, { type UseEmblaCarouselType } from 'embla-carousel-react';
+import { ArrowLeft, ArrowRight } from 'lucide-react';
```

Comment:
Any chance we could find an equalivate icon using `@fortawesome/react-fontawesome`? It would be nice to avoid adding another dependency if possible.

Reply:
Yeah, Let's do that. I will update the icons accordingly

Reply:
This has been changed